Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -56,7 +56,7 @@ public struct Configuration: Hashable {
                  indentation: IndentationStyle = .default) {
 
         if let pinnedVersion = swiftlintVersion, pinnedVersion != Version.current.value {
-            queuedPrintError("Currently running SwiftLint \(Version.current.value) but " +
+            queuedFatalError("Currently running SwiftLint \(Version.current.value) but " +
```

Comment:
This won't do what you want. This will crash SwiftLint and not print the message to the console in release builds.

You want to keep `queuedPrintError()` but `exit(2)` afterwards.